lib/plugins/aws/package/compile/events/stream/index.test.js

```diff
@@ -234,6 +234,35 @@ describe('AwsCompileStreamEvents', () => {
       ).to.equal(null);
     });
 
+    it('should not throw error if IAM role is referenced from cloudformation parameters', () => {
+      awsCompileStreamEvents.serverless.service.functions = {
+        first: {
+          role: { Ref: 'MyStreamArn' },
+          events: [
+            {
+              // doesn't matter if DynamoDB or Kinesis stream
+              stream: 'arn:aws:dynamodb:region:account:table/foo/stream/1',
```

Comment:
Shouldn't a `Ref` be used here as well so that we all cases covered or am I missing something 🤔

Reply:
This actually doesn't even test this functionality, it tests if `Ref` can be used with the role. I didn't intend to implement it but I accidentally did 😬 -> https://github.com/serverless/serverless/pull/6856/files#diff-f37cbc6a3ed6657ca5e07f484492cacdR156

Other tests are testing the stream arn `Ref` usage.